lib/rmq.service.ts

```diff
@@ -106,7 +106,7 @@ export class RMQService {
 				}
 				const { content } = msg;
 				if (content.toString()) {
-					this.logger.recieved(`[${topic}] ${content.toString()}`);
+					this.logger.debug(`[${topic}] ${content.toString()}`);
```

Comment:
I think the logger class should implement the received method. The transportation layer is not usual to debug thing and should have dedicated methods.

I suggest making a RMQLogger class, extending LoggerService and implementing these methods.